### PR TITLE
improved passbolt, added seafile

### PR DESCRIPTION
**Pull request by AmphoMoronal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ The http server is then listening on port 8000.
     - "/srv/traefik/certs:/certs"
   ```
 * Test traefik setup - I wrote it from what I remembered last time doing it...
-* Think about splitting the three [traefik container definition](./docs/installation/) into separate files (to avoid duplicate configuration fragments).
+* Think about splitting the three [traefik container definition](./docs/installation/20_traefik.md) into separate files (to avoid duplicate configuration fragments).
```
Comment (felbinger) [inline README.md:71]: nope - the folder contains the three definitions - not this one file

File: README.md [hunk 2]
```diff
@@ -97,14 +97,11 @@ The http server is then listening on port 8000.
 ### Services
 #### More
 * Monitoring
-* Matrix Bridges (Telegram, Signal)
+* Matrix Bridges (Signal)
```
Comment (felbinger) [inline README.md:100]: Why didn't you just do it right now? I mean you just set it up, want are you waiting for?

File: docs/services/seafile.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+# Seafile
+
+Seafile ist eine sichere, Open-Source-Plattform für die Synchronisation, Freigabe und Zusammenarbeit von Dateien, die
+sowohl eine On-Premises- als auch eine Cloud-basierte Bereitstellungsoption bietet.
+
+```yaml
+version: '3.9'
+
+services:
+  mariadb:
```
Comment (felbinger) [inline docs/services/seafile.md:10]: did you check if it is supporting nowadays postgresql too?

**Review by felbinger — CHANGES_REQUESTED** [review]

File: docs/services/seafile.md (hunk 3 above)
Comment (AmphoMoronal) [reply docs/services/seafile.md:10]: No. Blocks it.

Comment (felbinger) [reply docs/services/seafile.md:10]: Alright, so mariadb ist still required

File: docs/services/matrix.md [hunk 4]
```diff
@@ -425,3 +425,73 @@ Wenn der Container nun erneut neu gestartet wurde, kann man in seiner Matrix Ins
...
+```shell
+sudo docker compose exec postgres \
+  psql -U synapse -d synapse -c 'CREATE DATABASE "mautrix-signal";'
+```
+
+Anschließend sollten zumindest folgende Werte in der Datei `/srv/matrix/mautrix-signal/config.yaml` angepasst werden:
+
+```yaml
+homeserver:
+  # Hier muss angegeben werden, wie der mautrix-telegram
```
Comment (felbinger) [inline docs/services/matrix.md:450]: `mautrix-telegram` => `mautrix-signal`

File: docs/services/matrix.md [hunk 5]
```diff
@@ -425,3 +425,73 @@ Wenn der Container nun erneut neu gestartet wurde, kann man in seiner Matrix Ins
...
+  # Container den synapse homeserver innerhalb des matrix 
+  # docker netzwerks erreichen kann. Dies geschieht über 
+  # den Hostname des Containers und dem entsprechenden Port
+  address: http://synapse:8008
+  # Hier muss die domain des Homeservers angegeben werden
+  domain: domain.de  # hier steht bei euch etwas anderes
+
+appservice:
+  # Hier muss angegeben werden, wie der synapse
+  # container den mautrix-telegram bridge bot
```
Comment (felbinger) [inline docs/services/matrix.md:460]: `mautrix-telegram` => `mautrix-signal`

**Review by felbinger — CHANGES_REQUESTED** [review]

File: docs/services/matrix.md (hunk 5 above)
Comment (AmphoMoronal) [reply docs/services/matrix.md:460]: Yes comments are a thing...

**Review by felbinger — APPROVED** [review]

**Pull request merged**